### PR TITLE
Added the link to my UI Vesion

### PR DESCRIPTION
**Pull request by afeso**

### CODE REVIEW

File: JS101/fundamentals/rock-paper-scissors-project.md [hunk 1]
```diff
@@ -109,6 +109,10 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 - [vartanbeno's solution](https://github.com/vartanbeno/rock-paper-scissors) - [View in browser](https://vartanbeno.github.io/rock-paper-scissors/)
 - [Alexey Leonov's solution](https://github.com/axelerleo/rock-skissors-paper) - [View in browser](https://axelerleo.github.io/rock-skissors-paper/)
 - [Fabio Oliveira's solution](https://github.com/ffabiorj/rock_paper_scissors) - [View in browser](https://cdn.rawgit.com/ffabiorj/rock_paper_scissors/dab5e260/index.html)
+<<<<<<< HEAD
```
Comment (KevinMulhern) [inline JS101/fundamentals/rock-paper-scissors-project.md]: Hey @afeso, there seems to be a few merge conflict markers in this PR. Can you remove them please? 😄

File: JS101/fundamentals/rock-paper-scissors-project.md [hunk 2]
```diff
@@ -109,6 +109,10 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
...
+- [Benny Thai's solution](https://github.com/Deckins/rock_paper_scissors) - [View in browser](https://deckins.github.io/rock_paper_scissors/)
+- [Franklyn Afeso's solution](https://github.com/afeso/janken) - [View in browser](https://afeso.github.io/janken_ui)
+=======
```
Comment (KevinMulhern) [inline JS101/fundamentals/rock-paper-scissors-project.md]: heres another one

File: JS101/fundamentals/rock-paper-scissors-project.md [hunk 3]
```diff
@@ -157,3 +161,4 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 - [Pranshu's solutions](https://github.com/pranshugaba/rock_paper_scissors)
 - [Ol4er's solutions](https://github.com/ol4er/RPS)
 - [Allan Burns' solution](https://github.com/allanlburns/rock-paper-scissors/blob/master/rock_paper_scissors.html)
+>>>>>>> 27d588fedb1b34552b0dba3917a1d704596bd3cf
```
Comment (KevinMulhern) [inline JS101/fundamentals/rock-paper-scissors-project.md]: and the last one.

**Comment by afeso** [comment]
Ive made the changes to my file.  hopefully, that should remove  the conflicts

**Comment by KevinMulhern** [comment]
Perfect, thank you @afeso 

👍 LGTM

**Pull request merged**